Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -630,16 +630,16 @@ class TabTrayController: UIViewController {
     }
 
     private func openNewTab(request: NSURLRequest? = nil) {
-        toolbar.addTabButton.userInteractionEnabled = false
+        toolbar.userInteractionEnabled = false
```

Comment:
This `userInteractionEnabled` flag feels like a hack. Having a flag works around the bug, but why is this bug happening in the first place?